src/views/LegalNoticeView.vue

```diff
@@ -31,16 +31,15 @@ const navigateToHome = () => {
     </div>
     <div class="col-6">
       <Card>
-        <template #title> Kontakt </template>
+        <template #title> {{ $t('legalNotice.contact') }}</template>
         <template #content>
           <p>
             Tel.: +49 30 5019-2346<br />
```

Comment:
It’s not necessary to translate the text here, but it would be better to manage and maintain all text consistently in the JSON file for easier updates and uniformity.